Resources/doc/reference/configuration/proxy-client.rst

```diff
@@ -236,6 +237,32 @@ HTTP method for sending purge requests to the Symfony HttpCache. Make sure to
 configure the purge plugin for your HttpCache with the matching header if you
 change this.
 
+cloudflare
+-------
+
+.. code-block:: yaml
+
+    # config/packages/fos_http_cache.yaml
+    fos_http_cache:
+        proxy_client:
+            cloudflare:
+                zone_identifier: '<my-zone-identifier>'
+                authentication_token: '<user-authentication-token>'
+
+``authentication_token``
+"""""""""""""""""""""""
+
+**type**: ``string``
+
+User API token for authentication against Cloudflare APIs, requires Zone.Cache Purge permissions.
```

Comment:
```suggestion
User API token for authentication against Cloudflare APIs, requires ``Zone.Cache Purge`` permissions.
```

(is this the name of the permission? or is it `Zone.Cache` Purge?

Reply:
The latter, updated in dfc680ab9c45ed8d8d9d0eb9b26dfc44aebe0d96